### PR TITLE
Add cmake build support to travis CI

### PR DESCRIPTION
**Pull request by rhabacker**
See https://github.com/dlfcn-win32/dlfcn-win32/issues/92

### CODE REVIEW

**Review by traversaro — APPROVED** [review]
It seems fine for me, but let's see also @pali  opinion.

**Comment by pali** [comment]
Unbelievable! It is working. Last time I tried it, cmake build failed...

Just one cosmetic notice: On page https://travis-ci.org/github/dlfcn-win32/dlfcn-win32/builds/750998252 is order: For first 4 jobs is order: cmake then Makefile. And for the last two jobs is order Makefile then cmake. Could you switch order of last two jobs, so first would be also cmake?

**Review by pali — APPROVED** [review]
Perfect, you have fixed it!

**Review by traversaro — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,17 @@ env:
 
 jobs:
   include:
+    - dist: trusty
+      compiler: i586-mingw32msvc-gcc
+      addons:
+        apt:
+          packages:
+            - mingw32
+            - wine
```
Comment (pali) [inline .travis.yml:26]: Now I spotted that in packages is missing `cmake`. Job may start failing in future if (recycled) image would not contain it.

Comment (rhabacker) [reply .travis.yml:26]: Because this mr was already merged, the change needs to be submitted with a different mr.

**Pull request merged**